src/api/app/models/user.rb

```diff
@@ -612,7 +616,14 @@ def lock!
     end
   end
 
+  def delete
+    delete!
+  rescue APIException, ActiveRecord::RecordInvalid, ActiveXML::Transport::Error => _exception
```

Comment:
There is no point to store the exception here. Please remove that.

Also I don't think we should ignore any ActiveXML::Transport::Error or APIException that might occur. This might cover bugs.